static/usage/datetime/multiple/react.md

```diff
@@ -0,0 +1,20 @@
+```tsx
+import React, { useRef, useEffect } from 'react';
+import { IonDatetime } from '@ionic/react';
+
+function Example() {
+  const datetime = useRef<null | HTMLIonDatetimeElement>(null);
+
+  useEffect(() => {
+    if(!datetime.current) return;
+    datetime.current.value = ['2022-06-03', '2022-06-13', '2022-06-29'];
+  }, []);
```

Comment:
This is done to get around this issue, which also applies here: https://github.com/ionic-team/ionic-framework/issues/25041

Reply:
Should we update that issue to mention it will affect `ion-datetime` with `multiple`? Also, should we add a comment in the playground so in the future when that issue is resolved, we can update this example and remove this code?

Reply:
Good call 👀 I updated the issue and will add a tech debt ticket.

Reply:
Ticket made: https://ionic-cloud.atlassian.net/browse/FW-1872 I skipped adding the comment since I didn't want it to show up in the actual docs site (and putting it in `demo.html` felt weird when it's not relevant to that), but I think the Jira ticket should be good enough for tracking this.